cms/templates/widgets/header.html

```diff
@@ -36,12 +35,10 @@ <h1 class="branding"><a href="/">
             grading_url = reverse('contentstore.views.grading_handler', kwargs={'course_key_string': unicode(course_key)})
             advanced_settings_url = reverse('contentstore.views.advanced_settings_handler', kwargs={'course_key_string': unicode(course_key)})
             tabs_url = reverse('contentstore.views.tabs_handler', kwargs={'course_key_string': unicode(course_key)})
+            certificates_url = ''
```

Comment:
actually this header.html is rendered for multiple views. so it make sense to have building of URLs here instead of putting them in every view.